.gitignore

```diff
@@ -2,6 +2,8 @@
 # Created by https://www.gitignore.io/api/rust,jupyternotebooks,visualstudiocode
 # Edit at https://www.gitignore.io/?templates=rust,jupyternotebooks,visualstudiocode
 
+/
```

Comment:
why ?

Reply:
i don't known why it was there .. it's removed now